.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+Fixes issue #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]
```

Comment:
What if this doesn't fix an issue?

Reply:
so can one submit a PR without even referencing a issue or even if it doesn't has one issue for it?